src/baustein.js

```diff
@@ -366,6 +394,16 @@ export function isComponent(component) {
  */
 export function handleEvent(event, componentsChain) {
 
+    if (handlingEvent) {
+        pendingEvents.push(arguments);
+        return;
+    }
+
+    handlingEvent = true;
+
+    ensureEventHasMethod(event, 'stopPropagation', 'propagationStopped');
+    ensureEventHasMethod(event, 'preventDefault', 'defaultPrevented');
```

Comment:
![screen shot 2015-04-07 at 15 03 23](https://cloud.githubusercontent.com/assets/1671025/7024816/49f0ee22-dd37-11e4-831c-5ec907e43111.png)

:cry: 

I suppose we could polyfil it for IE, but will probably be quite complicated.